src/PattyServer.js

```
@@ -90,8 +92,7 @@ class PattyServer {
       gid = await Utils.getGID(this.options.processOwner);
       process.setgid(gid);
     } catch(e) {
-      // throw PattyError.other()
-      this.logError(`Could not change process owner group to "${gid}"`, e);
```

this is now before the init of the logger so we throw instead of logging